.github/workflows/ci-aarch64.yml

```
@@ -0,0 +1,148 @@
+#===============================================================================
+# Copyright contributors to the oneDAL project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+name: "CI AArch64"
+
+on:
+  push:
+    branches: [main, "rls*"]
+  pull_request:
+    types: [opened, synchronize, reopened]
```

```suggestion
on:
  pull_request:
    branches:
      - main
  push:
    branches:
      - main
```
This should align it with our regular CI, docker CI, and copyright checker workflows

---

done